lldb/test/API/lang/swift/unit-tests/Info.plist

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you delete all of these files on purpose?

Reply:
Yes, this test relies on the behavior that no longer works after this patch.